src/MaterialDesignControls/Implementations/CustomMaterial3/ContentBox.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Xamarin.Forms;
+
+namespace Plugin.MaterialDesignControls.Material3.Implementations
+{
+    public partial class ContentBox : Frame
```

Comment:
Same idea than a previous comment, move this features to the CustomFrame control that we should rename to MaterialCard.